config/schedule.yml

```diff
@@ -0,0 +1,4 @@
+send_push_notifications_job:
+  cron: '*/5 * * * *'
```

Comment:
can you remind me the syntax? :trollface:

Reply:
It's every 5 minute, but we still have to take a decision about this.